optking/opt_helper.py

```diff
@@ -51,11 +25,27 @@
 
 
 class Helper(ABC):
+    """
+    Base class for CustomHelper (accepts user provided gradients) and EngineHelper (uses MolSSI's QCEngine for gradients)
```

Comment:
Is it clear (or can we make it clear) what Helper one should use if one wants to use psi4 or qcengine to compute gradients, but then also wants to modify the gradients with something extra by hand.  I see how one could do that psi4 and customHelper; I'm less sure about other options.

Reply:
Yeah `CustomHelper` is what you want for modified gradients. Otherwise you'd need to create an `OptimizationManger` manually. I can add a statement to that effect in a few places. the getting started and optimizations pages might be good places. Anywhere else?

Reply:
We've got the L-J test already.  We can add additional examples as as they are developed for users, but ultimately a power-user will be responsible to figure out how to get their own gradient.